### ISSUE TITLE
something wrong with sync_batchnorm

### ISSUE THREAD

**Issue by HayashiKatsura** [THU-LYJ-Lab/dmt#9]
hi author 

could u pls send me the code about sync_batchnorm.py?
i found it was ignored at .gitignore.
so sincerely hope u could send the code file at my mail 221123120290@zjut.edu.cn,
thx so much

**Comment by datnguyen-tien204** [comment]
> hi author
> 
> could u pls send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn), thx so much

Do you get sync_batchnorm.py ? If you got, could you send me? Thank you. My mail: nguyentiendat_12422TN@utehy.edu.vn

**Comment by datnguyen-tien204** [comment]
> hi author
> 
> could u pls send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn), thx so much

I fixed successfully. If you need, please reply this comment.

**Comment by HayashiKatsura** [comment]
> > hi author
> > could u pls send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn), thx so much
> 
> I fixed successfully. If you need, please reply this comment.

Hello sir,

no ,i have not got this file, i'll appreciate if u could send this file to me (221123120290@zjut.edu.cn)， thx

**Comment by datnguyen-tien204** [comment]
> > > hi author
> > > could u pls send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn), thx so much
> > 
> > 
> > I fixed successfully. If you need, please reply this comment.
> 
> Hello sir,
> 
> no ,i have not got this file, i'll appreciate if u could send this file to me ([221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn))， thx

please check mail. I sent it.

**Comment by CYYJL** [comment]
> > hi author
> > could u pls send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [221123120290@zjut.edu.cn](mailto:221123120290@zjut.edu.cn), thx so much
> 
> Do you get sync_batchnorm.py ? If you got, could you send me? Thank you. My mail: [nguyentiendat_12422TN@utehy.edu.vn](mailto:nguyentiendat_12422TN@utehy.edu.vn)

Hi, hope you're doing well! I ran into the same issue and was wondering if I could ask you for your copy of sync_batchnorm.py. I'd really appreciate the help! (My e-mail: cyyjlpcgrad@gmail.com)

**Comment by CYYJL** [comment]
Here you can find the installation method for Synchronized-BatchNorm-PyTorch: https://github.com/EndlessSora/TSIT

**Comment by chunliwang52-prog** [comment]
hi author
could you please send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [250320040541@stu.haust.edu.cn](mailto:250320040541@stu.haust.edu.cn), thank you so much

**Comment by 6a6f454s5fd** [comment]
作者好，
能不能把sync_batchnorm.py的代码发给我？我发现它在.gitignore被忽视了。真心希望你能把代码文件寄到我[邮箱，2471279@stu.neu.edu.cn],非常感谢!!!

**Comment by datnguyen-tien204** [comment]
> 作者好， 能不能把sync_batchnorm.py的代码发给我？我发现它在.gitignore被忽视了。真心希望你能把代码文件寄到我[邮箱，[2471279@stu.neu.edu.cn](mailto:2471279@stu.neu.edu.cn)],非常感谢!!!

[sync_batchnorm.py](https://github.com/user-attachments/files/24284309/sync_batchnorm.py)

**Comment by datnguyen-tien204** [comment]
> hi author could you please send me the code about sync_batchnorm.py? i found it was ignored at .gitignore. so sincerely hope u could send the code file at my mail [250320040541@stu.haust.edu.cn](mailto:250320040541@stu.haust.edu.cn), thank you so much

[sync_batchnorm.py](https://github.com/user-attachments/files/24284314/sync_batchnorm.py)
